clarifai/utils/logging.py

```diff
@@ -357,3 +359,7 @@ def format(self, record):
           default=self.json_default,
           cls=self.json_cls,
       )
+
+
+# the default logger for the SDK.
+logger = get_logger(logger_level=os.environ.get("LOG_LEVEL", "INFO"), name="clarifai")
```

Comment:
Will the user be able to see the module logging the log line e.g. `compute_cluster` etc ?

Reply:
we'll see the filename and line number still.